buffalo/cmd/soda_generators.go

```diff
@@ -19,11 +20,25 @@ func newSodaGenerator() *gentronics.Generator {
 	f.Should = should
 	g.Add(f)
 
-	c := gentronics.NewCommand(goGet("github.com/markbates/pop/..."))
-	c.Should = should
-	g.Add(c)
+	g.Add(&gentronics.Func{
+		Should: should,
+		Runner: func(rootPath string, data gentronics.Data) error {
+			var driver string
+			if data["dbType"] == "sqlite3" {
```

Comment:
just a thought, but if we drop the `_driver` from the build tag then we don't need this if statement.

also, it might be possible that people have two different DBs, say SQLite in development or test and Postgres in production. That's more of a thing we need to worry about during `buffalo build`. For that we can look at the `database.yml` file during build time and grab all of the dialects being used there to pass as flags.